### ISSUE TITLE
Mention in warnings that README contents are used if description field is missing

### ISSUE THREAD

**Issue by s-burris** [kornelski/cargo-deb#99]
In `manifest_check_config()` in [manifest.rs](https://github.com/kornelski/cargo-deb/blob/1c45759d58424311c09ccfd902dab138ebac1f08/src/manifest.rs#L903), a warning is displayed if the `description` is not set in `Cargo.toml`.

Currently this appears as `warning: description field is missing in Cargo.toml` but I recommend including info about the usage of README's contents such as: `warning: description field is missing in Cargo.toml, using README.md text instead`

Why? Non-public codebases may have READMEs that contain information not intended for public consumption. Currently, `cargo-deb` will include it in deb packages quietly.

Additionally, `README.md` contents typically appear long and ugly (in e.g. `apt info`), so I believe a warning is warranted in the case of public codebases as well.

**Comment by drauschenbach** [comment]
Also FWIW when a README is used because the description field is blank, it may end with a blank line, causing an unwanted section break which corrupts the Packages file.

```sh
$ apt-get update
...
E: Encountered a section with no Package: header
E: Problem with MergeList /var/lib/apt/lists/__REDACTED__/_dists_jammy_main_binary-amd64_Packages
E: The package lists or status file could not be parsed or opened.
```

**Comment by kornelski** [comment]
This is not correct. If the description is missing it's replaced by "generated from Rust crate X".
